src/common/run_command.cpp

```diff
@@ -6,16 +6,59 @@
 
 #include <common/run_command.h>
 
+#include <logging.h>
 #include <tinyformat.h>
 #include <univalue.h>
+#include <util/string.h>
 
-#ifdef ENABLE_EXTERNAL_SIGNER
+#ifdef ENABLE_SUBPROCESS
```

Comment:
I think you can just drop these `#ifdef`s around our own headers. Same for in `RunCommandParseJSON` below.

Reply:
Wasn't sure about this. One reason for setting `ENABLE_SUBPROCESS=OFF` in the first place would be that the operating system doesn't support subprocesses. As subprocess is a header-only library, including the `subprocess.hpp` header would mean use of `fork` `execv` `CreateProcessW` etc. So it could break the compile.

Reply:
If we don't care about supporting such operating systems (only iOS comes to mind), it would imo be preferable to remove the option entirely.